### ISSUE TITLE
A11y testing and support

### ISSUE THREAD

**Issue by matthew-chirgwin** [strimzi/strimzi-ui#20]
The Strimzi-ui should be fully accessible (a11y) from day 0. This issue is to investigate what tooling we can use to catch a11y issues early/make sure the UI is accessible to any user of it.

As per the proposal (https://github.com/strimzi/proposals/pull/6), we will target the superset of US Section 508 and WCAG21AA rulesets.

**Comment by nictownsend** [comment]
https://www.npmjs.com/package/cypress-accessibility-checker (if not already aware)
